core/infrastructure/marketplace-api-client-adapter/mock-adapters/contributor-client-adapter-mock.ts

```
@@ -0,0 +1,25 @@
+import { ContributorStoragePort } from "@/core/domain/contributor/outputs/contributor-storage-port";
+
+import { mockHttpStorageResponse } from "../http/mock-http-client/mock-http-storage-response";
+
+export class ContributorClientAdapterMock implements ContributorStoragePort {
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)